kernel/dev/rtc/rtc.cpp

```diff
@@ -87,10 +100,10 @@ void read_rtc() {
     rtc_year = rtc_get_register(0x09);
     rtc_century = rtc_get_register(RTC_CURRENT_CENTURY);
 
-    while  ((last_second != rtc_second) || (last_minute != rtc_minute) ||
-            (last_hour != rtc_hour)     || (last_day != rtc_day)       ||
-            (last_month != rtc_month)   || (last_year != rtc_year)     ||
-            (last_century != rtc_century))
+    while ((last_second != rtc_second) || (last_minute != rtc_minute) || 
```

Comment:
I'm going to go ahead and approve this since this doesn't really need to be fixed, but trailing whitespace should be trimmed. I'm not sure if there's a vim plugin for that (I'm almost certain there is though), but I would recommend installing one. That way extra whitespace is trimmed away when you save the file.